.github/workflows/push.yml

```diff
@@ -13,12 +13,24 @@ on:
     # seed the build cache.
     branches:
       - main
+  schedule:
+    - cron: '0 0,12 * * *'  # Runs at 00:00 and 12:00 UTC daily
 
 env:
   GOTESTSUM_FORMAT: github-actions
 
 jobs:
+  cleanups:
+    runs-on: ubuntu-latest
```

Comment:
This should run on the hosted runner, as it is performing a write against the repository.

Reply:
Updated.

Reply:
For future reference, this is what happens on ubuntu-latest:
> Run gh cache delete --all --repo databricks/cli || true
HTTP 403: Although you appear to have the correct authorization credentials, the `databricks` organization has an IP allow list enabled, and your IP address is not permitted to access this resource. (https://api.github.com/repos/databricks/cli/actions/caches?per_page=100)